contrib/bash/cjdns-trap.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+PIPE_NAME="client-core-$(tr -cd 'a-f0-9' < /dev/urandom | head -c 8)"
```

Comment:
Wonder if it'd be nicer to use `openssl rand -hex 8` instead of shelling out to two utils?